graph/config/cytoscape/cytoscape.go

```diff
@@ -130,28 +132,39 @@ func NewConfig(trafficMap graph.TrafficMap, o graph.ConfigOptions) (result Confi
 
 	buildConfig(trafficMap, &nodes, &edges, o)
 
-	// Add compound nodes as needed
-	switch o.GroupBy {
-	case graph.GroupByApp:
-		if o.GraphType != graph.GraphTypeService {
-			groupByApp(&nodes)
-		}
-	case graph.GroupByVersion:
-		if o.GraphType == graph.GraphTypeVersionedApp {
-			groupByVersion(&nodes)
```

Comment:
No more grouping by version?
...although I really can't recall seeing a graph where nodes are grouped by version

Reply:
Right, it was an unused/unnecessary feature that I had added way back in the beginning.  Simplifying things by removing it now.